lib/peep.rb

```diff
@@ -0,0 +1,39 @@
+require 'pg'
+
+class Peep
+
+  attr_reader :id, :message, :username, :posted_at
+
+  def initialize(id:, message:, username:, posted_at:)
+    @id  = id
+    @message = message 
+    @username = username 
+    @posted_at = posted_at
+  end
+
+    def self.all
+        if ENV['ENVIRONMENT'] == 'test'
```

Comment:
This if else statement is going to be required in any function in this class and its already adding a lot of bloat just being used twice, could this be extracted into its own function?